src/pyOpenMS/pxds/MzMLSqliteHandler.pxd

```diff
@@ -19,28 +19,66 @@ cdef extern from "<OpenMS/FORMAT/HANDLERS/MzMLSqliteHandler.h>" namespace "OpenM
         MzMLSqliteHandler(MzMLSqliteHandler &) nogil except + # compiler
 
         void readExperiment(MSExperiment & exp, bool meta_only )  nogil except +
+            # wrap-doc:
+                #   Read an experiment into an MSExperiment structure
+                #   -----
+                #   :param exp: The result data structure
+                #   :param meta_only: Only read the meta data
   
         void readSpectra(libcpp_vector[MSSpectrum] & exp, libcpp_vector[int] indices, bool meta_only ) nogil except +
+            # wrap-doc:
+                #   Read an set of spectra (potentially restricted to a subset)
+                #   -----
+                #   :param exp: The result
+                #   :param indices: A list of indices restricting the resulting spectra only to those specified here
+                #   :param meta_only: Only read the meta data
 
         void readChromatograms(libcpp_vector[MSChromatogram] & exp, libcpp_vector[int] indices, bool meta_only ) nogil except +
+             wrap-doc:
+                #   Read an set of spectra (potentially restricted to a subset)
+                #   -----
+                #   :param exp: The result
+                #   :param indices: A list of indices restricting the resulting spectra only to those specified here
+                #   :param meta_only: Only read the meta data
   
-        Size getNrSpectra() nogil except +
+        Size getNrSpectra() nogil except + # wrap-doc:Get number of spectra in the file, reutrns the number of spectra
   
-        Size getNrChromatograms() nogil except +
+        Size getNrChromatograms() nogil except + # wrap-doc:Returns the number of chromatograms in the file
   
         void setConfig(bool write_full_meta, bool use_lossy_compression, double linear_abs_mass_acc)  nogil except +
+            # wrap-doc:
+                #   Set file configuration
+                #   -----
+                #   :param write_full_meta: Whether to write a complete mzML meta data structure into the RUN_EXTRA field (allows complete recovery of the input file)
+                #   :param use_lossy_compression: Whether to use lossy compression (ms numpress)
+                #   :param linear_abs_mass_acc: Accepted loss in mass accuracy (absolute m/z, in Th)
+                #   :param sql_batch_size: Batch size of SQL insert statements
```

Comment:
TODO: Check what is up with that parameter, since it is not represented in the wrapper.